README.md

```diff
@@ -7,7 +7,26 @@
 
 This repository contains a lightweight client for an [AHBicht](https://github.com/Hochfrequenz/ahbicht) powered backend by Hochfrequenz.
 
-## How to use this Repository on Your Machine
+## How to use the Client
+- Install using pip:
+```bash
+pip install ahbichtfunctionsclient
+```
+Then call it
+
+```python
+from ahbicht.mapping_results import PackageKeyConditionExpressionMapping
+from ahbichtfunctionsclient import HochfrequenzPackageResolver
+from maus.edifact import EdifactFormat, EdifactFormatVersion
+
+package_resolver = HochfrequenzPackageResolver()
+package_resolver.edifact_format = EdifactFormat.UTILMD
+package_resolver.edifact_format_version = EdifactFormatVersion.FV2204
+package_mapping = await package_resolver.get_condition_expression("10P")
+assert isinstance(package_mapping, PackageKeyConditionExpressionMapping)
```

Comment:
Vielleicht hier noch etwas mehr demo code, mit koomentaren damit am ersten blick ersichtlich wird was da genau passiert?

Reply:
was da genau passiert kann man in den unnittest ablesen. kommentare kommen...

Reply:
https://github.com/Hochfrequenz/ahbicht-functions-python-client/pull/6/commits/d4e19d8b752de0cfe295132844116515383858e6 kommentare

Reply:
https://github.com/Hochfrequenz/ahbicht-functions-python-client/pull/6/commits/ff4e306b2a4467ec80774461d91a8578d0131277 <-- ein kommentar der mit "100% code coverage" impliziert, dass man sehr genau nachvolziehen kann, was passiert :)